### PR TITLE
Database

### PR DESCRIPTION
**Pull request by Angelica137**
our little db

### CODE REVIEW

File: client/src/component/Rooms.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ let socket;
 const Rooms = ({ setRoom, username, setUsername }) => {
   const [userRoom, setUserRoom] = useState("");
   const [roomList, setRoomList] = useState([]);
-  const ENDPOINT = "http://localhost:5000";
+  const ENDPOINT = process.env.ENDPOINT;
```
Comment (FayeCarter) [inline client/src/component/Rooms.js:9]: Nice

File: database.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+//mongo "mongodb+srv://whatsstack-dpcwj.mongodb.net/test" --username WhatStack;
+
+const mongoose = require("mongoose");
+const connection =
+  "mongodb+srv://WhatStack:faye15cool@whatsstack-dpcwj.mongodb.net/test";
```
Comment (FayeCarter) [inline database.js:5]: password still showing

File: index.js [hunk 3]
```diff
@@ -21,15 +23,30 @@ io.on("connection", (socket) => {
     if (!roomList.includes(room)) {
       roomList.push(room);
     }
-
     socket.join(room);
     console.log(name);
+
+    const q = Message.find({ room });
```
Comment (FayeCarter) [inline index.js:29]: const q?

File: index.js [hunk 4]
```diff
@@ -21,15 +23,30 @@ io.on("connection", (socket) => {
...
+
+    q.then((res) => io.sockets.in(room).emit("messages", res));
+    //console.log(test.messages);
```
Comment (PhilipVigus) [inline index.js:32]: No need for comment here

File: database.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+//mongo "mongodb+srv://whatsstack-dpcwj.mongodb.net/test" --username WhatStack;
```
Comment (PhilipVigus) [inline database.js:1]: Delete this too pls

**Pull request merged**